src/main/webapp/WEB-INF/jsp/product-list.jsp

```diff
@@ -7,28 +8,53 @@
 </head>
 
 <body>
+<div>
     <div>
-        <div>
-            <h2>Shop</h2>
-        </div>
+        <h2>Shop</h2>
     </div>
+</div>
+<div>
     <div>
-        <div>
-            <table>
+        <table>
+            <tr>
+                <th>Product Name</th>
+                <th>Price PLN</th>
+                <th></th>
+            </tr>
+            <c:forEach var="tempProduct" items="${products}">
                 <tr>
-                    <th>Product Name</th>
-                    <th>Price PLN</th>
-                    <th>Add To Basket</th>
+                    <td> ${tempProduct.productName}</td>
+                    <td> ${tempProduct.price}</td>
+                    <td>
+                        <input type="button" onclick="addToCart(${tempProduct.productId})" value="Add to cart"/>
+                    </td>
                 </tr>
-                <c:forEach var="tempProduct" items="${products}">
-                    <tr>
-                        <td> ${tempProduct.productName}</td>
-                        <td> ${tempProduct.price}</td>
-                        <td>add</td>
-                    </tr>
-                </c:forEach>
-            </table>
-        </div>
+            </c:forEach>
+
+        </table>
+    </div>
+</div>
+<div>
+    <div>
+        <button id="addToCart" onclick="submitForm()">Cart</button>
     </div>
+</div>
+<form:form id="idForm" method="post" modelAttribute="formIdList" action="/shop/cart">
+    <input type="hidden" id="idInput" name="productIds">
+</form:form>
 </body>
+<script>
```

Comment:
Script is good, but it's not a good practice to place it with HTML. Please extract it to a separate file.